cmd/kube-apiserver/app/server.go

```diff
@@ -430,7 +430,7 @@ func buildGenericConfig(
 	// Since not every generic apiserver has to support protobufs, we
 	// cannot default to it in generic apiserver and need to explicitly
 	// set it in kube-apiserver.
-	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
+	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = kubeoptions.DefaultStorageMediaType
```

Comment:
it messes up api content type and storage content type

Reply:
Thanks a lot! Updated.